website/static/style.css

```
@@ -120,253 +160,787 @@ pre {
+  display: block;
+  margin-bottom: 12px;
+  color: var(--ink);
+  font-size: 11px;
+}
+
+.runbook-steps code {
+  display: block;
+  color: var(--ink-soft);
+  font-size: 12px;
+  line-height: 1.45;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)